easybuild/easyblocks/p/petsc.py

```diff
@@ -193,7 +193,7 @@ def configure_step(self):
 
             deps = [dep['name'] for dep in self.cfg.dependencies() if not dep['name'] in depfilter]
             for dep in deps:
-                if type(dep) == str:
+                if isinstance(dep, str):
```

Comment:
```python
                if isinstance(dep, string_type):
```

(requires `from easybuild.tools.py2vs3 import string_type` to be added on top as well)